ext/nnstreamer/tensor_filter/tensor_filter_caffe2_core.h

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (C) 2019 Samsung Electronics Co., Ltd. All rights reserved.
+ * Copyright (C) 2019 HyoungJoo Ahn <hello.ahn@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ */
+/**
+ * @file   tensor_filter_caffe2_core.h
+ * @author HyoungJoo Ahn <hello.ahn@samsung.com>
+ * @date   05/31/2019
+ * @brief  connection with caffe2 libraries.
+ *
+ * @bug     No known bugs.
+ */
+#ifndef TENSOR_FILTER_CAFFE2_CORE_H
+#define TENSOR_FILTER_CAFFE2_CORE_H
+
+#include "nnstreamer_plugin_api_filter.h"
+
+#ifdef __cplusplus
+#include <iostream>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wredundant-decls"
```

Comment:
Question: ignored "-Wredundant-decls",  is it necessary to avid build warning in caffe2?

Reply:
it's removed